.changeset/hot-falcons-grab.md

```diff
@@ -0,0 +1,13 @@
+---
+"@fluidframework/datastore-definitions": minor
+"@fluidframework/runtime-definitions": minor
+---
+---
+"section": other
+---
+
+The timing of the `op` event on `IFluidDataStoreRuntimeEvents` and `IContainerRuntimeBaseEvents` has changed.
```

Comment:
The headers/titles (first line of the changeset contents) usually skip the trailing stop. The rephrasing is pretty subjective, it just felt more "title-y" to me. Feel free to keep as is and just remove the period.
```suggestion
Change when the `op` event on `IFluidDataStoreRuntimeEvents` and `IContainerRuntimeBaseEvents` is emitted
```